### PR TITLE
Add gn flags to generate size info for the supersize tool.

### PR DESCRIPTION
**Pull request by zhongqiliang**
b/382770170

### CODE REVIEW

File: cobalt/build/configs/common.gn [hunk 1]
```diff
@@ -15,3 +15,5 @@ ffmpeg_branding = "Chrome"
 # Removing these flags causes a lot of compiler errors in starboard code.
 clang_use_chrome_plugins = false
 enable_check_raw_ptr_fields = false
+
+optimize_for_size = true
```
Comment (niranjanyardi) [inline cobalt/build/configs/common.gn:19]: what's the default ? also just from the name doesn't seem like we want this for debug/devel builds - might affect how much debug information is present in the binary/ debug logging etc

Comment (zhongqiliang) [reply cobalt/build/configs/common.gn:19]: https://chromium.googlesource.com/chromium/src/+/d1a04b008747abf9231b676ad1b34e53bad43973/build/config/compiler/BUILD.gn#74
It is defined here, but I do not know if it exists in our build system.

Comment (zhongqiliang) [reply cobalt/build/configs/common.gn:19]: oh, great, thanks.

**Review by niranjanyardi — APPROVED** [review]
apart from open comment lgtm

File: cobalt/android/BUILD.gn [hunk 2]
```diff
@@ -203,4 +203,5 @@ content_shell_apk_tmpl("cobalt_apk") {
   ]
   shared_libraries = [ ":libcobalt_content_shell_content_view" ]
   command_line_flags_file = "content-shell-command-line"
+  include_size_info = is_official_build
```
Comment (andrewsavage1) [inline cobalt/android/BUILD.gn:206]: I just want to confirm that this does not impact the cobalt apk any way and so is fine to include in the official (release) builds

Comment (zhongqiliang) [reply cobalt/android/BUILD.gn:206]: chrome does the same thing.

**Review by andrewsavage1 — APPROVED** [review]

**Pull request merged**